apps/scraper-runner/src/main.ts

```diff
@@ -0,0 +1,79 @@
+import * as scholarsScrape from '@foodmedicine/scholars-scraper';
+import * as articleParser from '@foodmedicine/article-parser';
+import * as healthSiteScraper from '@foodmedicine/health-site-scraper';
+import * as fs from 'fs';
+import {
+  ParsedArticleParagraphStandalone,
+  ParsedArticle,
+  ParsedArticleParagraph,
+} from '@foodmedicine/interfaces';
+
+/**
+ * Get the top percentage of an array
+ * @param arr an array sorted in descending order
+ * @param percent percentage of items to be returned
+ */
+function getTopPercentage(arr: any[], percent = 5): any[] {
+  return arr.slice(0, Math.floor((arr.length * percent) / 100));
+}
+
+async function main() {
```

Comment:
You have these `main` functions everywhere, maybe a holdover from your work with C/C++?  It's fine if you prefer this style, but you can also just call this stuff outside of a function

Reply:
I choose to use main because I wanted to have a way to easily call each package in case that they would not to be strung together  in some series for the final code. Also, have a main function allows me to use async await

Reply:
👍